waspc/data/Generator/templates/react-app/src/entities/_entity/components/CreateForm.js

```diff
@@ -116,7 +207,20 @@ export default class {= formName =} extends React.Component {
           </div>
           {=/ showSubmitButton =}
         </form>
+      {=# isXRayModeEnabled =}
+        </>
+      </EntityFormInfoBox>
+      {=/ isXRayModeEnabled =}
       </div>
     )
   }
 }
+
+{=# isXRayModeEnabled =}
+export default connect(state => ({
```

Comment:
Same as for other parts, if you connect EntityInfoFormBox to state directly, you will not have to do this connect here and you will also not need all the imports and you will not need to have conditional "default" export, so this whole file can be simplified.

Reply:
great idea, done, connected InfoBox directly to Redux.